pkg/v1/remote/options.go

```diff
@@ -89,6 +89,7 @@ var retryableStatusCodes = []int{
 	http.StatusBadGateway,
 	http.StatusServiceUnavailable,
 	http.StatusGatewayTimeout,
+	http.StatusTooManyRequests,
```

Comment:
I'd prefer to make this change as a separate PR. Retrying in the event of rate limits is a complex subject, and automatically retrying here can make problems worse instead of better.

Reply:
Ok, I will move this change to a separate PR.

Reply:
Done with #1636 and #1635